trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/ColumnIndices.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits;
+
+/**
+ * Provides setters focused on configuring a validator to look at particular
+ * column within a row of a provided file
+ */
+trait ColumnIndices {
+
+  /**
+   * Sets the array of indices for which the validator to do its checks on.
```

Comment:
I think more detail explaining this would be helpful. What do you think of this?
```suggestion
   * Sets the array of indices for the validator to do its checks on.
   *
   * Specifically, in this context, an index is the key of the column the
   * validator should check. In the case of lists, this index/key is a sequential
   * integer created by PHP and in the case of associative arrays, this index/key
   * will likely be a meaningful string (such as names of column headers).
```